assets/scss/layouts/footer/_footer.scss

```diff
@@ -144,17 +144,29 @@
 }
 
 .footer-payment-icons {
+    display: flex;
+    flex-wrap: wrap;
     height: remCalc(40);
+    justify-content: center;
+    list-style-type: none;
+    margin-left: 0;
     margin-top: spacing("single");
 
+    @include breakpoint("small") {
+        justify-content: normal;
+    }
+
     @include breakpoint("medium") {
         margin-top: spacing("double");
     }
 
+    .footer-payment-item {
+        height: 100%;
+    }
+
```

Comment:
two questions about that:
1. why we need to set the height of `footer-payment-item`?
2. if we need - can we use `align-items: stretch;`for parent?